standards/core/store-sync.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WAKU-STORE-SYNC
+name: Waku Store Synchronization
+editor: Simon-Pierre Vivier <simvivier@status.im>
+contributors:
+---
+
+## Abstract
+
+This document describe the strategy Waku Store node will employ to stay synchronized.
+The goal being that all store nodes eventually archive the same set of messages.
+
+## Background / Rationale / Motivation
+
+Message propagation in the network is not perfect,
+even with GossipSub mechanisms to detect missed messages.
+Nodes can also go offline for various reason outside our control.
+Store nodes that want to provide a good service must be able to remedy situations like these.
+By having store nodes syncronize with each other through various protocols,
+the set of archived messages network wide will be eventually consistent.
+
+## Theory / Semantics
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Various protocols and features that help with message consistency are described below.
+
+### Store Resume
+
+This feature allow a node to fill the gap in messages for the period it was last offline.
+At startup, a node use the Store protocol to query a random node for
+the time interval since it was last online.
+Messages returned my the query are then added to the node local archive storage.
+It is RECOMMENDED” to limit the time interval to a maximum of 6 hours.
+
+### Waku Sync
+
+Nodes that stay online can still miss messages.
+[Waku Sync](TODO link to spec) is the protocol used to find those messages by periodically syncing with random nodes.
+It is RECOMMENDED to trigger a sync every 5 minutes for a time range of the last hour.
```

Comment:
I'd recommend separating the mechanism/principle from the implementation detail into two separate statements:
Nodes SHOULD trigger the store sync procedure regularly enough to maintain close to real-time consistency.
We RECOMMEND that sync is triggered every 5 minutes over the time range of the last hour.

Reply:
What is the current RECOMMENDED number of nodes to sync with every 1 hour?